examples/cryptocurrency/README.md

```diff
@@ -0,0 +1,58 @@
+# Cryptocurrency Tutorial
+
+[![Build status][travis-image]][travis-url]
+[![Gitter][gitter-image]][gitter-url]
+
+[travis-image]: https://img.shields.io/travis/exonum/cryptocurrency.svg?style=flat-square
+[travis-url]: https://travis-ci.org/exonum/cryptocurrency
```

Comment:
Urls are wrong. Perhaps just remove the badges from the readme?

Reply:
I'm not sure. On the one hand, we still have badges in other readme files. On the other hand, it is probably enough to have them in the root readme.

@slowli What do you think?

Reply:
I don't have a particularly strong opinion on this topic. IMO, any of the two options would do; the badges allow to quickly check CI status, but they are slightly misleading because CI tasks are not split along with packages.